packages/pn-commons/src/hooks/useDownloadDocument.ts

```diff
@@ -23,10 +24,10 @@ type Props = {
 // -------------------------------
 // Carlos Lombardi and Andrea Cimini, 2022.11.17
 // -------------------------------
-export function useDownloadDocument({ url, clearDownloadAction }: Props) {
+export function useDownloadDocument({ url, isMobile, clearDownloadAction }: Props) {
   useEffect(() => {
```

Comment:
Try to use the hook isMobile, to remove the isMobile property

Reply:
I've already tried the solution proposed and the result is an application crash. This is mainly caused by using a hook inside a regular function instead of a functional component.